pom.xml

```diff
@@ -154,39 +145,86 @@ since this is impossible for AWS S3 dependencies without Maven.
-                Update version number below.
-                Note that code below doesn't download netcdfAll (as hoped)
-                  but it does identify that the jar should be included in mvn compile.
-                  (mvn compile says "File already exist, skipping".)
-              Alternative that didn't work for me: 
-                This approach is from Jennifer Oxelson. See https://github.com/oxelson/netcdfAll/blob/main/pom.xml 
-                I think this isn't needed/doesn't work(?). 
-            - ->
+            <!-- https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-antrun-plugin -->
             <plugin>
-                <groupId>com.googlecode.maven-download-plugin</groupId>
-                <artifactId>download-maven-plugin</artifactId>
-                <version>1.6.8</version>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.1.0</version>
                 <executions>
                     <execution>
-                        <phase>validate</phase>
+                        <?m2e execute onConfiguration,onIncremental?>
+                        <id>download-netcdf</id>
+                        <phase>initialize</phase>
                         <goals>
-                            <goal>wget</goal>
+                            <goal>run</goal>
                         </goals>
                         <configuration>
-                            <url>https://artifacts.unidata.ucar.edu/repository/downloads-netcdf-java/5.5.2/netcdfAll-5.5.2.jar</url>
-                            <outputFileName>netcdfAll-5.5.3.jar</outputFileName>
-                            <outputDirectory>${project.basedir}/external_libs</outputDirectory>
+                            <target>
+                                <get src="https://downloads.unidata.ucar.edu/netcdf-java/5.5.3/netcdfAll-5.5.3.jar" dest="${project.basedir}/WEB-INF/lib/netcdfAll-5.5.3.jar" skipexisting="true" verbose="true"/>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <?m2e execute onConfiguration?>
+                        <id>download-erddap-content</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <get src="https://github.com/BobSimons/erddap/releases/download/${erddapcontent.download.version}/erddapContent.zip" dest="${project.basedir}/erddapContent.zip" skipexisting="true" verbose="true"/>
+                                <!-- note: the overwrite="false" option doesn't seem to act as advertised.  It will overwrite files in the dest folder if they are older than in archive -->
```

Comment:
When I last ran this, I'm pretty sure it did not overwrite my local setup.xml file (even though it had not been edited more recently than the most [recent commit](https://github.com/BobSimons/erddapContent/commit/711ecd9791ae5543e909c29351d71f900bf4eda6) on erddapContent of setup.xml.  So, I'd say remove this warning here and on the lines below unless you or others happen to notice the overwrite behavior.  In practice, it really probably only matters for setup.xml and other customized files in content/erddap directory (such as datasets.xml) since the other ref files below are aren't likely to be modified.